stdlib/2and3/asyncore.pyi

```diff
@@ -15,6 +15,7 @@ from errno import (EALREADY, EINPROGRESS, EWOULDBLOCK, ECONNRESET, EINVAL,
 # cyclic dependence with asynchat
 _maptype = Dict[str, Any]
 
+socket_map: _maptype = ...  # Undocumented
```

Comment:
Doesn't this map filenos (i.e. `int`s) to something? `_maptype` uses `str` as key.

Reply:
I based typing on existing stubs for `loop()` and similar functions.

Source:
```python3
def loop(timeout=30.0, use_poll=False, map=None, count=None):
    if map is None:
        map = socket_map
    ....
```
Stub:
```python3
_maptype = Dict[str, Any]

def loop(timeout: float = ..., use_poll: bool = ..., map: _maptype = ..., count: Optional[int] = ...) -> None: ...
```

Reply:
I guess `_maptype` is wrong then. Could you change it to `Dict[int, Any]`?